src/components/ConfirmDialog.tsx

```diff
@@ -21,20 +21,20 @@ interface ConfirmDialogProps {
   dataTestId?: string;
 }
 
-const ConfirmDialog: FC<ConfirmDialogProps> = ({
+const ConfirmDialog = ({
   children,
   open,
   title,
   onAccept,
   onCancel,
   isLoading = false,
   dataTestId,
-}) => {
+}: ConfirmDialogProps) => {
   const { t } = useTranslation('common');
 
   return (
-    <Dialog open={open} onClose={onCancel} data-testid={dataTestId}>
-      <DialogTitle>{title}</DialogTitle>
+    <Dialog open={open} onClose={onCancel} data-testid={dataTestId} PaperProps={{ 'aria-labelledby': 'titleId' }}>
+      <DialogTitle id="titleId">{title}</DialogTitle>
```

Comment:
blir tittelen `titleId` eller `{title}` her?

Reply:
Det blir noe sånn som  `<label id="titleID">` i DOM, så betyr `'aria-labelledby': 'titleId'` at verdien til `titleId` skal være tittel for skjermleser.
Så skjermleseren sier bare det som står i `{title}`. Har aldri prøvd skjermleser da, så det burde vi gjøre...